tools/IoTEdgeDevOps/IoTEdgeDashboard/.config/dotnet-tools.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this file is automatically added by VS, which is introduced in .net core 3.1

Reply:
Do we need it? If not, we can remove it.

Reply:
Leave it as-is; otherwise I believe it will add to the project again when open in VS2019.